include/spm/rel/machi.h

```diff
@@ -2,9 +2,11 @@
 
 #include <common.h>
 #include <evt_cmd.h>
-
+#include <spm/evtmgr.h>
 CPP_WRAPPER(spm::machi)
 
+USING(spm::evtmgr::EvtScriptCode)
+
```

Comment:
The changes to this file are probably no longer neccessary now that you've moved the cooking evt